src/main/java/kr/motd/maven/os/Detector.java

```diff
@@ -143,4 +161,86 @@ private static String normalize(String value) {
         }
         return value.toLowerCase(Locale.US).replaceAll("[^a-z0-9]+", "");
     }
+
+    /**
+     * Gets a collection of OSes that the given detected OS is "like". The {@code detectedName} will
+     * always appear in the collection.
+     */
+    private static Collection<String> getOsLike(String detectedName) {
+        String[] osReleaseFileNames = {"/etc/os-release", "/usr/lib/os-release"};
```

Comment:
I was talking about moving the String[] into a static final instance in the class ;)